mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/JPAMessageMapper.java

```diff
@@ -93,6 +94,12 @@ public Flux<MessageUid> listAllMessageUids(Mailbox mailbox) {
             .map(MailboxMessage::getUid);
     }
 
+    @Override
+    public Flux<MailboxMessage> findInMailboxReactive(Mailbox mailbox, MessageRange messageRange, FetchType ftype, int limitAsInt) {
+        return Flux.defer(Throwing.supplier(() -> Flux.fromIterable(findAsList(mailbox.getMailboxId(), messageRange, limitAsInt))).sneakyThrow())
```

Comment:
Shouldn't internal `Flux` (`Flux.fromIterable(findAsList())`) have same `Schedulers.boundedElastic()`?

Reply:
No it is already running on this scheduler.